### PR TITLE
Fix vulnerabilities in application-connectivity

### PR DESCRIPTION
**Pull request by rafalpotempa**
**Description**

Changes proposed in this pull request:

- Upgarde k8s clients in connection-token-handler

**Related issue(s)**

### CODE REVIEW

File: resources/application-connector/values.yaml [hunk 1]
```diff
@@ -32,35 +32,35 @@ global:
...
+    version: "PR-10651"
   application_registry_tests:
     version: "d556963d"
   application_broker:
     version: "52f52cc9"
   application_connectivity_certs_setup_job:
-    version: "960b1536"
+    version: "PR-10651"
   application_connectivity_validator:
     version: "56b8cb60"
```
Comment (koala7659) [inline resources/application-connector/values.yaml:65]:
```suggestion
    version: "PR-10651"
```

File: components/application-registry/go.mod [hunk 2]
```diff
@@ -25,6 +25,9 @@ replace (
 	github.com/containerd/containerd => github.com/containerd/containerd v1.4.1
```
Comment (koala7659) [inline components/application-registry/go.mod:25]: I don't see go.sum file for application-registry. Shouldn't be it also here?

**Review by koala7659 — DISMISSED** [review]
Please check my comment and suggestion

**Review by koala7659 — APPROVED** [review]
Everything is fine

**Review by akgalwas — APPROVED** [review]

**Pull request merged**